src/main/java/com/unityTest/courseManagement/entity/AssignmentEnrollment.java

```diff
@@ -30,10 +34,10 @@
 	private int id;
```

Comment:
Add a `@JsonIgnore` annotation to this id field. We don't need the ids returned by the endpoint.